packages/main/types/responses.ts

```diff
@@ -79,8 +79,28 @@ export interface EnhancedGenerateContentResponse
  * @public
  */
 export interface GenerateContentResponse {
+  /** Candidate responses from the model. */
   candidates?: GenerateContentCandidate[];
+  /**
+   * A set of the feedback metadata the prompt specified in
```

Comment:
I know this is from the proto, but it's kind of garbled, maybe try using the proto comment next to the property which I think is more clear: https://source.corp.google.com/piper///depot/google3/google/ai/generativelanguage/v1main/generative_service.proto;l=444